product_docs/docs/pgd/5/cli/using_cli.mdx

```diff
@@ -52,50 +53,65 @@ p-vjljj303dk-a-3 3521351376 p-vjljj303dk-a witness ACTIVE        ACTIVE       Up
-		"route_priority": -1,
-		"route_fence": false,
-		"route_writes": true,
-		"route_reads": true,
-		"route_dsn": "host=p-vjljj303dk-a-1-node.vmk31wilqpjeopka.biganimal.io user=streaming_replica sslmode=verify-full port=5432 sslkey=/controller/certificates/streaming_replica.key sslcert=/controller/certificates/streaming_replica.crt sslrootcert=/controller/certificates/server-ca.crt application_name=p-vjljj303dk-a-1 dbname=bdrdb"
-	},
-...
+  {
+    "node_name": "kaftan",
+    "node_group_name": "dc1_subgroup",
+    "node_kind_name": "data",
+    "join_state": "ACTIVE",
+    "node_status": "Up",
+    "node_id": 3490219809,
+    "node_seq_id": 2,
+    "node_local_dbname": "bdrdb"
+  },
+  {
+    "node_name": "kaolin",
+    "node_group_name": "dc1_subgroup",
+    "node_kind_name": "data",
+    "join_state": "ACTIVE",
+    "node_status": "Up",
+    "node_id": 2111777360,
+    "node_seq_id": 1,
+    "node_local_dbname": "bdrdb"
+  },
+  {
+    "node_name": "kaboom",
+    "node_group_name": "dc1_subgroup",
+    "node_kind_name": "data",
+    "join_state": "ACTIVE",
+    "node_status": "Up",
+    "node_id": 2710197610,
+    "node_seq_id": 3,
+    "node_local_dbname": "bdrdb"
+  }
 ]
 ```
 
 The PGD CLI supports the following output formats.
 
 | Setting | Format | Considerations |
 | ------- | ------ | --------- |
-|    none    | Tabular | Default format. This setting presents the data in tabular form.|
-| `json`    | JSON | Presents the raw data with no formatting. For some commands, the JSON output might show more data than the tabular output, such as extra fields and more detailed messages. |
-| `yaml`    | YAML | Similar to the JSON output but as YAML and with the fields ordered alphabetically. Experimental and might not be fully supported in future versions. |
+|  none    | Tabular | Default format. This setting presents the data in tabular form.|
+| `json`   | JSON | Presents the raw data with no formatting. For some commands, the JSON output might show more data than the tabular output, such as extra fields and more detailed messages. |
+| `ascii`  | ASCII | Presents the data in a ascii table for another tabular option. |
+| `psql`   | PSQL | Another tabular view in PSQL format. |
+| `modern` | Tabular | Another tabular view very similar to the PSQL format option, but with a more explicit table. |
+| `markdown-table` | Markdown | Another tabular option in Markdown style. |
+| `default` | Tabular | Same as using no option, as it is the default format. |
 
 ## Accessing the command line help
 
 To list the supported commands, enter:
 
 ```sh
-pgd help
+pgd --help
 ```
 
 For help with a specific command and its parameters, enter `pgd help <command_name>`. For example:
```

Comment:
The `pgd help <command_name>` is not a valid syntax

Reply:
got it